testnets/wavehashtestnet/assetslist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "wavehashtestnet",
+    "assets": [
+      {
+        "description": "WaveHash A Permissioned Blockchain",
+        "denom_units": [
+          {
+            "denom": "uwahax",
+            "exponent": 0
+          },
+          {
+            "denom": "wahax",
+            "exponent": 6
+          }
+        ],
+        "base": "uwaha",
+        "name": "WaveHash",
+        "display": "wavehash",
+        "symbol": "WAHAX",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/WaveHashProtocol/chain-registry/master/testnets/wavehashtestnet/images/WaveHash.png",
+          "svg": "https://raw.githubusercontent.com/WaveHashProtocol/chain-registry/master/testnets/wavehashtestnet/images/WaveHash.svg"
```

Comment:
the svg seems to be a tiled conversion from the PNG--probably not worth including, in my opinion